rosbag2_transport/CMakeLists.txt

```
@@ -463,6 +463,18 @@ if(BUILD_TESTING)
   endfunction()
   call_for_each_rmw_implementation(test_record_services_for_rmw_implementation)
 
+  ament_add_gmock_executable(test_record_topic_types test/rosbag2_transport/test_record_topic_types.cpp)
+  target_link_libraries(test_record_topic_types rosbag2_transport ${test_msgs_TARGETS} rosbag2_test_common::rosbag2_test_common)
+  function(test_record_services_for_rmw_implementation)
+    set(rmw_implementation_env_var RMW_IMPLEMENTATION=${rmw_implementation})
+
+    ament_add_gmock_test(test_record_topic_types
+      TEST_NAME test_record_topic_types${target_suffix}
+      ENV ${rmw_implementation_env_var}
+    )
+  endfunction()
+  call_for_each_rmw_implementation(test_record_services_for_rmw_implementation)
```

Mistake from copy-past. Shall be:
```suggestion
  call_for_each_rmw_implementation(test_record_topic_types_for_rmw_implementation)
```